protocols/direct-fund/direct-fund.go

```diff
@@ -228,7 +228,8 @@ func (s DirectFundObjective) safeToDeposit() bool {
 		chainHolding, ok := s.C.OnChainFunding[asset]
 
 		if !ok {
-			return false
+			// If there are no holdings on chain we use 0 for our calculations
+			chainHolding = big.NewInt(0)
```

Comment:
I think there may be a slightly better approach to fix behaviour, which I will make a small PR for.

Reply:
#220 -- see what you think @lalexgap